atlasexec/atlas.go

```diff
@@ -394,24 +408,10 @@ func (c *Client) runCommand(ctx context.Context, args []string) (io.Reader, erro
 	cmd.Stderr = &stderr
 	cmd.Stdout = &stdout
 	if err := cmd.Run(); err != nil {
-		switch {
-		case stderr.Len() > 0:
-			// Atlas CLI writes the error to stderr.
-			// So this's critical issue, return the error.
-			return nil, &cliError{
-				summary: strings.TrimSpace(stderr.String()),
-				detail:  strings.TrimSpace(stdout.String()),
-			}
-		case cmd.ProcessState.ExitCode() == 1:
-			// When the exit code is 1, it means that the command
-			// failed but the output is still valid JSON.
-			//
-			// `atlas migrate lint` returns 1 when there are
-			// linting errors.
-		default:
-			// When the exit code is not 1, it means that the
-			// command wasn't executed successfully.
-			return nil, err
+		return nil, cliError{
+			summary:  strings.TrimSpace(stderr.String()),
+			detail:   strings.TrimSpace(stdout.String()),
+			exitCode: cmd.ProcessState.ExitCode(),
```

Comment:
Why does it matter what the exit code is? Isn't cli error always returning exit code != 0

Reply:
can be removed, originally written when I thought lint error = 1 and the rest of the errors are other codes (the code previously had a special case for exit code 1)